### ISSUE TITLE
Create slot / bookings autofix

### ISSUE THREAD

**Issue by ikusteu** [eisbuk/EisBuk#882]
Continuing from #881 and similar to slot / attendance check / autofix, we should create an autofix function that reads the latest check report (of performs a fresh check) and applies fixes with respect to found discrepancies.

This ticket is opened as a reminder, but I feel we should discuss the appropriate autofix actions after the check ticket is completed, so most of it is **TBD**

**Comment by fadwamahmoud** [comment]
- slots are the source of truth so bookedSlots are deleted/updated not slots
- stray bookings should be deleted
- we need to agree on whether to update or delete bookedSlots with mismatched dates/intervals 

- Simplest approach: 
Delete all mismatches from bookedSlots
ie:
  ```
    "xEnpZQjUKPd6TODnNcvr--ed0503a3-c9ce-4563-ad9b-63cdf139a3d7": {
    "collection": "bookedSlots",
    "exists": true,
    "date": "2022-06-04",
    "intervals": "09:40-11:20"
  }
  
  
delete `bookings/ed0503a3-c9ce-4563-ad9b-63cdf139a3d7/bookedSlots/xEnpZQjUKPd6TODnNcvr`
    
@ikusteu

**Comment by ikusteu** [comment]
Yes, I would go with deleting the faulty bookings as a first pass. @silviot what are your thoughts on that?

**Comment by ikusteu** [comment]
I think, however, that it would be good to send out the emails for deleted bookings (so that customers know their bookings were removed). 
Or...send emails to admins only and have them handle the confirmations with the customers.

**Comment by ikusteu** [comment]
Fixed in #911
